src/main/kotlin/no/nav/helse/spion/domene/Periode.kt

```diff
@@ -5,4 +5,6 @@ import java.time.LocalDate
 data class Periode(
     val fom: LocalDate,
     val tom: LocalDate
-)
\ No newline at end of file
+) {
```

Comment:
Pirking og synsing: Kunne vurdere å ekstrahere ut parsingen i en egen metode for å bevare enkelheten i data classen. Ikke viktig.

Reply:
Ja dette var bare for at det skulle bli penere kode i repo-koden ved mapping fra databasen